README.md

```diff
@@ -788,6 +855,25 @@ RSpec.configure do |config|
 end
 ```
 
+### Elasticsearch client options
+
+All connection options, except the `:prefix`, are passed to the `Elasticseach::Client.new` ([chewy/lib/chewy.rb](https://github.com/toptal/chewy/blob/f5bad9f83c21416ac10590f6f34009c645062e89/lib/chewy.rb#L153-L160)):
+
+```ruby	
```

Comment:
Since you already placed a link to source code, I don't think we need to copy it here (actually better not to, so we don't need to worry on sync this).